docs/tutorials/discord-slash-command.md

```diff
@@ -0,0 +1,149 @@
+---
+sidebar_position: 4
+---
+
+# Discord slash command
+
+In this article, I will show you how to use **slash commands** to interact with your Discord bot based on an external web service. Specifically, this flow function is a weather inquiry bot. When you type `/weather` + the city name, the bot will respond to you with the weather information of this city.
+
+![](discord-slash-command.png)
```

Comment:
Since we cannot see the argument to the slash command in the UI, I think you need to modify the app to display the city name in the response.